pandas/tests/indexing/test_indexing.py

```diff
@@ -666,6 +666,20 @@ def test_index_type_coercion(self):
                     idxr(s2)['0'] = 0
                     assert s2.index.is_object()
 
+    @pytest.mark.parametrize("index,val", [
```

Comment:
can you add a comment with the issue number

Reply:
these tests should be in pandas/tests/indexes/test_base (look for other contains tests)

Reply:
`test_index_not_contains` and `test_index_contains` are in the same file a few lines above, should I move these up or create pandas/tests/indexes/test_base and move them there?